scripts/rollup.config.js

```diff
@@ -75,6 +75,14 @@ const options = {
     return defaults;
   }
 };
+
+if (process.env.CI_TEST_TYPE) {
+  if (process.env.CI_TEST_TYPE === 'unit') {
+    options.testInput = ['!test/playback.test.js', 'test/**/*.test.js'];
```

Comment:
if running unit tests build everything except for playback tests. Otherwise only run playback tests.